Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PostCampaignPledgeRewardsSummaryViewController.swift

```diff
@@ -44,6 +44,8 @@ final class PostCampaignPledgeRewardsSummaryViewController: UIViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.view.isHidden = true
+    self.pledgeTotalViewController.view.isHidden = true
```

Comment:
Note: It's important to never put anything in `bindStyles` that can be overwritten later. Anything to do with initial state needs to go in `viewDidLoad` instead. I spent hours trying to debug why the table wasn't showing up on the checkout screen 😭

Reply:
Thanks for calling that out! For some reason, the separator view was still showing... I just hid that as well and the table is back to being fully hidden before data loads!